docs/getting-started/get-departures.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Get Departures
+---
+
+## Goal
+
+In this step, we will download plane departures data from the REST-Interface described in the previous step using Smart Data Lake Builder.
+
+## Config File
+
+With Smart Data Lake Builder, you describe your data-pipelines in a config-file using the [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md) format.
+You can also have split up your configuration accross multiple files, but in this Part, we will only use one file.
+
+A data pipeline is composed of two entities: *dataObjects* and *actions*.
+
+An action transforms one (or multiple) data_object into another (or multiple) data_objects.
+In every data pipeline, you will have at least one *dataObject* for your input and one for your output.
+If you have more than one action, you will also have at least one *dataObject* for each intermediary step between two actions.
+
+In our case, in order to get our departures data, we are going to build one action. Hence we need one dataObject for our input, and one for our output.
+Create a directory called config and in that directory, create an empty file called application.conf. This is where we will define our data pipeline.
+
+## Define departures objects
+Add this to the file:
+
+    dataObjects {
+      ext-departures {
+        type = WebserviceFileDataObject
+        url = "https://opensky-network.org/api/flights/departure?airport=LSZB&begin=1630200800&end=1630310979"
+      }
+      stg-departures {
+        type = JsonFileDataObject
+        path = ext-departures
```

Comment:
path should be stg-departures, but you could use {~id} here.

Reply:
Done in https://github.com/pgruetter/sdl-docs/pull/1/commits/580d9323dfdc73120449c99dd1120e3b96b56324